antenna-selection/README.rst

```diff
@@ -1,8 +1,8 @@
 Demo of Antennas Selection
 ===========================
-This code was taken from the webinar, *Quantum Programming with the Ocean Tools Suite*.
+This code was taken from the webinar, *Quantum Programming with the Ocean Tools Suite [2]_.
```

Comment:
The second asterisk is missing. Hence, `Quantum Programming with ... Suite` is no longer italicised.